pkg/controller/forwarding/forwarding_controller.go

```diff
@@ -89,40 +89,54 @@ func (r *ReconcileForwarding) Reconcile(request reconcile.Request) (reconcile.Re
 			// Return and don't requeue
 			return reconcile.Result{}, nil
 		}
-		// Error reading the object - requeue the request.
-		return reconcile.Result{}, err
+		// Error reading the object -- requeue
+		return reconcileResult, err
```

Comment:
So when you want the request to be requeued, you return `reconcileResult`, and when you do not want the result to be requeued, you return `reconcile.Result{}`?

Reply:
yes, `reconcileResult` has a field `RequeueAfter` that puts this back on the event queue